R/build-home-index.R

```diff
@@ -158,10 +158,13 @@ data_home_sidebar_links <- function(pkg = ".") {
   repo <- cran_link(pkg$package)
   links <- config_pluck(pkg, "home.links")
 
+  bug_reports <- pkg$desc$get_field("BugReports", default = NULL)
+  if (grepl("@", bug_reports)) bug_reports <- paste0("mailto:", bug_reports)
```

Comment:
I think you also need to check that `bug_reports` doesn't start with `http` because `@` is a valid sign in URLs.

Nitpick: Important statements like assignments should be on their own line, e.g.:

```suggestion
  if (grepl("@", bug_reports)) {
    bug_reports <- paste0("mailto:", bug_reports)
  }
```